dcos-log/router/middleware.go

```diff
@@ -1,10 +1,64 @@
 package router
 
-import "net/http"
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gorilla/mux"
+)
 
 func authMiddleware(next http.Handler) http.Handler {
+	client := &http.Client{
```

Comment:
might be nice to have `client` passed in as a param for `authMiddleware`, that way someone else could configure the client for you and you just use what you're given to execute calls against servers in the cluster